e2e-tests/content-strings/fields/insurance/your-buyer/index.js

```diff
@@ -3,4 +3,7 @@ export const YOUR_BUYER_FIELD_IDS = {
   BUYER_COUNTRY_LABEL: 'Country',
   BUYER_ORGANISATION: 'organisation',
   BUYER_ORGANISATION_LABEL: "Buyer's company or organisation name",
+  BUYER_ADDRESS: 'address',
+  BUYER_ADDRESS_LABEL: 'Address',
```

Comment:
We are having comment/ conversation every time.  These are actual; filed values declared in test and its base declaration test.Its lot of confusing. where do we need to bring the fields..

Reply:
We have to raise/resolve this because it wasn't addressed in the last PR.

We want to keep field IDs, separate from content strings for the field.

There is some information about this / the "why" in the README https://github.com/UK-Export-Finance/exip/tree/main-application#keeping-page-and-fields-separated

Here are some example controllers with a `pageVariables` function, that consumes field IDs and field content strings, ready to be consumed in the get/post controllers and nunjucks template:
- https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/controllers/insurance/policy-and-export/single-contract-policy/index.ts#L37
- https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/controllers/insurance/policy-and-export/about-goods-or-services/index.ts#L31

Example of field content string definitions for "policy and export": https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/content-strings/fields/insurance/policy-and-exports/index.ts - the first field does have an ID which is wrong, this is tech debt, All the other fields are how we want it to be.

Making this change is easier to do now rather than later, as the "your buyer" page adds more form fields and there are a lot of fields.

We want different areas of the codebase to be consistent and not deviating. This helps developers to digest the different areas and make changes, without having different approaches in different places.